e2e/src/templates/default/index.mjs

```diff
@@ -1,9 +1,10 @@
 import http from 'node:http';
+import fs from 'node:fs';
 
 const server = http.createServer((req, res) => {
   if (req.url === '/' || req.url === '/index.html') {
     res.writeHead(200, { 'Content-Type': 'text/html' });
-    res.end('Index page');
+    res.end(fs.readFileSync('./index.html', 'utf-8'));
```

Comment:
On `main` there's now a `template: file-server` that does this. Originally this default one was doing it also but it was reverted to minimal no-fs-operations in https://github.com/stackblitz/tutorialkit/pull/241#discussion_r1724743673.

Reply:
Ah nice! I'll use that one :+1: